Fix `passphrase should expire in in_memory_pass_phrase_session_length` mock test issue

This test sometimes fails with below error.
Need to fix.

>   ✖ user@passphrase-session-length-client-configuration.flowcrypt.test - passphrase should expire in in_memory_pass_phrase_session_length [ALL RETRIES FAILED for user@passphrase-session-length-client-configuration.flowcrypt.test - passphrase should expire in in_memory_pass_phrase_session_length]
>     ℹ (attempt 1 of 3) Retrying: AssertionError: expected undefined to equal '1234'
>     ℹ (attempt 2 of 3) Retrying: AssertionError: expected undefined to equal '1234'
>     ℹ (attempt 3 of 3) Failed:   /home/semaphore/git/flowcrypt-browser/test/source/tests/setup.ts:853
>             expect(savedPassphrase).to.be.equal(passphrase);

>       AssertionError: expected undefined to equal '1234'
>           at <anonymous> (/home/semaphore/git/flowcrypt-browser/test/source/tests/setup.ts:853:37)
>           at <anonymous> (/home/semaphore/git/flowcrypt-browser/test/source/test.ts:76:7)
>           at <anonymous> (/home/semaphore/git/flowcrypt-browser/test/source/browser/browser-pool.ts:144:61)
>           at <anonymous> (/home/semaphore/git/flowcrypt-browser/test/source/test.ts:71:5)
>           at <async> <anonymous> (/test/source/tests/setup.ts:832:180)
>           at <async> <anonymous> (../test/source/test.ts:71:53)
>           at <async> <anonymous> (/test/source/browser/browser-pool.ts:144:49)

--- a/test/source/mock/backend/backend-data.ts
+++ b/test/source/mock/backend/backend-data.ts
@@ -97,7 +97,7 @@ export class BackendData {
         "flags": [
           "FORBID_STORING_PASS_PHRASE",
         ],
-        "in_memory_pass_phrase_session_length": 3
+        "in_memory_pass_phrase_session_length": 10
       };
     }
     if (domain === 'no-submit-client-configuration.flowcrypt.test') {

--- a/test/source/tests/setup.ts
+++ b/test/source/tests/setup.ts
@@ -937,7 +937,7 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
         'cryptup_userpassphrasesessionlengthclientconfigurationflowcrypttest_rules'
       ]);
       let savedPassphrase = await BrowserRecipe.getPassphraseFromInMemoryStore(settingsPage, acctEmail, longid);
-      expect((rules as { in_memory_pass_phrase_session_length: number }).in_memory_pass_phrase_session_length).to.be.equal(3);
+      expect((rules as { in_memory_pass_phrase_session_length: number }).in_memory_pass_phrase_session_length).to.be.equal(10);
       expect(savedPassphrase).to.be.equal(passphrase);
       // wait 1 minute because we delete expired passphrase every 1 minute
       await Util.sleep(3);

--- a/test/source/tests/setup.ts
+++ b/test/source/tests/setup.ts
@@ -940,7 +940,7 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       expect((rules as { in_memory_pass_phrase_session_length: number }).in_memory_pass_phrase_session_length).to.be.equal(10);
       expect(savedPassphrase).to.be.equal(passphrase);
       // wait 1 minute because we delete expired passphrase every 1 minute
-      await Util.sleep(3);
+      await Util.sleep(10);
       savedPassphrase = await BrowserRecipe.getPassphraseFromInMemoryStore(settingsPage, acctEmail, longid);
       expect(savedPassphrase).to.be.undefined;
     }));

--- a/test/source/tests/setup.ts
+++ b/test/source/tests/setup.ts
@@ -939,7 +939,6 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       let savedPassphrase = await BrowserRecipe.getPassphraseFromInMemoryStore(settingsPage, acctEmail, longid);
       expect((rules as { in_memory_pass_phrase_session_length: number }).in_memory_pass_phrase_session_length).to.be.equal(10);
       expect(savedPassphrase).to.be.equal(passphrase);
-      // wait 1 minute because we delete expired passphrase every 1 minute
       await Util.sleep(10);
       savedPassphrase = await BrowserRecipe.getPassphraseFromInMemoryStore(settingsPage, acctEmail, longid);
       expect(savedPassphrase).to.be.undefined;

